test/clj/puppetlabs/trapperkeeper/services/webrouting/webrouting_service_test.clj

```diff
@@ -15,9 +15,11 @@
                      with-app-with-config]]
             [puppetlabs.trapperkeeper.testutils.logging
              :refer [with-test-logging]]
-            [schema.core :as schema]))
+            [schema.core :as schema]
+            [schema.test :as schema-test]))
 
 (use-fixtures :once ks-test-fixtures/with-no-jvm-shutdown-hooks)
+(use-fixtures :once schema-test/validate-schemas)
```

Comment:
You don't need a separate call to `use-fixtures`, you can just add `schema-test/validate-schemas` to the one above.